bin/gonzales.js

```diff
@@ -47,7 +47,7 @@ function processInputData(input) {
         process.stdout.write(ast.toJson());
         process.exit(0);
     } catch (e) {
-        if (!options.silent) process.stderr.write(e.message);
+        if (!options.silent) process.stderr.write(e.toString());
```

Comment:
Sure, sorry. It prints the ParseError stack message that shows the problematic line rather than just a terse error message, which made it a lot easier to debug.